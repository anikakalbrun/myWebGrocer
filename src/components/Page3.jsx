import React from 'react';

import Wrapper from './Wrapper';
import '../style/css/Page.css';

export default function Page3() {
    return (
        <Wrapper innerContainerClass='inverse'>
            <h1 className='presentation__title'>Thank You</h1>
        </Wrapper>
    );
}