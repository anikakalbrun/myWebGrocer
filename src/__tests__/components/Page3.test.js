import React from 'react';

import Page3 from '../../components/Page3';

let wrapper;

describe('Page3', () => {

    beforeEach(() => {
        wrapper = shallow(<Page3/>);
    });

    it('renders the Wrapper component', () => {
        expect(wrapper.find('Wrapper').length).toEqual(1)
    });

    it('passes the innerContainerClass to the Wrapper component', () => {
        expect(wrapper.find('Wrapper').at(0).props().innerContainerClass).toEqual('inverse')
    });

    it('renders proper h1', () => {
        expect(wrapper.find('Wrapper').at(0).props().children).toMatchSnapshot()
    });
});
