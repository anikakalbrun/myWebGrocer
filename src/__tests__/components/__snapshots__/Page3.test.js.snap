// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page3 renders proper h1 1`] = `
<h1
  className="presentation__title"
>
  Thank You
</h1>
`;
